app/src/main/java/org/fossasia/susi/ai/activities/ForgotPasswordActivity.java

```diff
@@ -150,15 +151,15 @@ public void onClick(DialogInterface dialog, int id) {
                     builder.setTitle(R.string.email_invalid_title);
                     builder.setMessage(R.string.email_invalid)
                             .setCancelable(false)
-                            .setPositiveButton("Retry", null);
+                            .setPositiveButton(getApplicationContext().getString(R.string.retry), null);
                     AlertDialog alert = builder.create();
                     alert.show();
                     Button pbutton = alert.getButton(DialogInterface.BUTTON_POSITIVE);
                     pbutton.setTextColor(Color.RED);
 
                 } else {
                     AlertDialog.Builder builder = new AlertDialog.Builder(ForgotPasswordActivity.this);
-                    builder.setTitle(response.code() + " Error");
+                    builder.setTitle(response.code() + R.string.error);
```

Comment:
Here the output will be `40462543`, where `62543` is the ID of the string `error` and 404 the response code. Change to getString please